File: src/main/java/com/github/zametki/model/VerificationRecordId.java
package com.github.zametki.model;

import com.github.mjdbc.DbMapper;
import com.github.mjdbc.Mapper;

public final class VerificationRecordId extends AbstractId {

    public VerificationRecordId(int id) {
        super(id);
    }

    @Mapper
    public static final DbMapper<VerificationRecordId> MAPPER = r -> new VerificationRecordId(r.getInt(1));
}

File: src/main/java/com/github/zametki/model/VerificationRecordId.kt
package com.github.zametki.model

import com.github.mjdbc.DbMapper
import com.github.mjdbc.Mapper

class VerificationRecordId(id: Int) : AbstractId(id) {
    companion object {

        @Mapper
        @JvmField
        val MAPPER = DbMapper { r -> VerificationRecordId(r.getInt(1)) }
    }
}
